pkgs/applications/video/vlc/default.nix

```diff
@@ -80,170 +21,50 @@
 
 let
   inherit (lib) optionalString optional optionals;
-in
-stdenv.mkDerivation rec {
-  pname = "${optionalString onlyLibVLC "lib"}vlc";
   version = "3.0.18";
 
-  src = fetchurl {
-    url = "http://get.videolan.org/vlc/${version}/vlc-${version}.tar.xz";
-    sha256 = "sha256-VwlEOcNl2KqLm0H6MIDMDu8r7+YCW7XO9yKszGJa7ew=";
+  srcs = rec {
```

Comment:
`srcs` and `meta` should be declared separately for the two derivations: otherwise, `linux.nix` would still be evaluable on darwin, and thus ibtool errors.

Reply:
Do I have to declare the entirety of meta separately? Or can I declare it once and then append the platforms field?

I've now declared srcs separately and the platforms for meta (but not descriptions, and etc) separately but I still get the same ibtool error.

Reply:
Oh, I checked the expressions in nixpkgs, and I think I'm wrong. Sorry for the inconvenience! It should be OK to inherit both of them. Let's see if ofborg evals.